src/commands/codepush/patch.ts

```diff
@@ -0,0 +1,102 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp, shortName, longName, required, hasArg, position, name } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest, clientCall } from "../../util/apis";
+import * as _ from "lodash";
+import * as chalk from "chalk";
+import * as semver from "semver";
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:patch");
+
+@help("Update the metadata for an existing release")
+export default class PatchCommand extends AppCommand {
+  
+  @help("CodePush deployment name.")
+  @required
+  @name("ExistingDeploymentName")
+  @position(0)
+  public deploymentName: string;
+
+  @help("Label of the release to update. Defaults to the latest release within the specified deployment.")
+  @required
+  @name("ExistingReleaseLabel")
+  @position(1)
+  public releaseLabel: string;
+
+  @help("Specifies whether this release should be considered mandatory. Putting -m flag means mandatory.")
+  @shortName("m")
+  public isMandatory: boolean;
+
+  @help("Specifies whether this release should be immediately downloadable. Putting -x flag means disabled.")
+  @shortName("x")
+  public isDisabled: boolean;
+
+  @help("Semver expression that specifies the binary app version(s) this release is targeting (e.g. 1.1.0, ~1.2.3).")
+  @shortName("t")
+  @longName("targetBinaryVersion")
+  @hasArg
+  public targetBinaryRange: string;
+
+  @help("Description of the changes made to the app with this release.")
+  @shortName("d")
+  @longName("description")
+  @hasArg
+  public description: string;
+
+  @help("Percentage of users this release should be immediately available to. This attribute can only be increased from the current value.")
+  @shortName("r")
+  @longName("rollout")
+  @hasArg
+  public rollout: string;
+
+  constructor(args: CommandArgs) {
+    super(args);
+  }
+
+  async run(client: MobileCenterClient): Promise<CommandResult> {
+
+    const app = this.app;
+    let release: models.LiveUpdateRelease;
+
+    if (this.targetBinaryRange === null && this.isDisabled === null && this.isMandatory === null && this.description === null && this.rollout === null) {
+      return failure(ErrorCodes.Exception, "At least one property must be specified to patch a release.");
+    }
+
+    if (this.rollout != null && this.rollout !== undefined) {
```

Comment:
`this.rollout != null` this condition already includes check for undefined, so there is no need use `&& this.rollout !== undefined`

Reply:
@christav I changed back to the double equal syntax since only using triple equal syntax without checking undefined will cause the issue. (default command without rollout parameter)